test/test_va_api_fixture.cpp

```diff
@@ -51,7 +55,7 @@ VAAPIFixture::VAAPIFixture()
 VAAPIFixture::~VAAPIFixture()
 {
     m_vaDisplay = NULL;
-    if (m_drmHandle >= 0)
+    if (m_drmHandle >= 0 && m_drmHandle != VAAPIFixtureSharedDisplay::s_drmHandle)
```

Comment:
I'm not sure I see how this can ever be false: m_drmHandle != VAAPIFixtureSharedDisplay::s_drmHandle. That is, m_drmHandle will never equal s_drmHandle. Or did I miss something?

Reply:
Used to be possible, but other cleanups fixed that.  Dropped.